contrib/cirrus/setup.sh

```diff
@@ -28,3 +28,14 @@ if [[ "$CIRRUS_TASK_NAME" == "build_cross" ]]; then
 	# TODO: We can move this to quay.io if we reach rate-limits, hopefully that's not gonna happen for netavark
 	echo '  "rustembedded/cross" = "docker.io/rustembedded/cross"'  >> /etc/containers/registries.conf.d/000-shortnames.conf
 fi
+
+req_env_vars AARDVARK_DNS_URL
+
+set -x  # show what's happening
+curl --fail --location -o /tmp/aardvark-dns.zip "$AARDVARK_DNS_URL"
+mkdir -p /usr/libexec/podman
+cd /usr/libexec/podman
+rm aardvark-dns
```

Comment:
This will fail if the file isn't there.  Maybe do `rm -f aardvark-dns*` to be sure.